### PR TITLE
excerpt can be now returned without surround <p>

### PR DESCRIPTION
**Pull request by ainthek**
please consider this as an enhancement, sometimes we need the excerpt as text (or inner HTML without the surrouding <p>", I have added optional option.selector, which can have now only one value
innerHTML, if specified, it does return innerHTML, if not it returns the same as before.

Thanx

### CODE REVIEW

**Comment by brunoscopelliti** [comment]
Hi there mantainers.
Thank you for your work.

I was considering exactly this question, and eventually proposing a pull request for this feature... 
Pretty strange considering this was opened 11h ago :)

I think we need this pull request merged because there is an inconsistency here:
when we get the excerpt from the front matter we get plain text; otherwise when there isn't the "excerpt" prop in the front matter we get the text surrounded by the `<p> ...</p>` html.

I've not a particular preference between the two versions, but consistency is definitely something important.

Let me know if is there something i can do to help.

**Comment by dpisklov** [comment]
Is there anyone looking after this repo? I really need this PR too...

File: lib/index.js [hunk 1]
```diff
@@ -27,11 +28,13 @@ function plugin(options){
...
         return; // don't mutate data
       }
-
-      debug('storing excerpt: %s', file);
       var $ = cheerio.load(data.contents.toString());
       var p = $('p').first();
-      data.excerpt = $.html(p).trim();
+      
+      data.excerpt = options && options.selector==='innerHTML'? $(p).html():$.html(p) 
+      data.excerpt && data.excerpt.trim();
```
Comment (Ajedi32) [inline lib/index.js]: I'm pretty sure this line doesn't actually do anything. String#trim doesn't mutate the string, it just returns a new string with the outer whitespace removed. Strings in JS are immutable.

**Comment by webketje** [comment]
Will be accounted for in next release see https://github.com/metalsmith/excerpts/issues/42

**Pull request closed without merging**